docs/gh_docker_workflow.md

```diff
@@ -0,0 +1,32 @@
+# Github workflow to build and push images to GCR
+
+## Executive summary
+
+The provided text outlines a workflow designed to build and push an image to the GitHub Container Registry. It involves creating three tags: pull request number, pull request name, and commit SHA.
+
+## Glossary
+
+- **Image:** The term "image" typically refers to a packaged, standalone software component that includes everything needed to run an application, such as the code, runtime, libraries, and dependencies.
+
+- **GitHub Container Registry (GCR):** GCR is a service provided by GitHub that allows users to store, manage, and distribute Docker container images within the GitHub ecosystem. It serves as a centralized repository for container images associated with GitHub repositories.
+
+- **GitHub Action:** A GitHub Action is essentially a workflow or automated process defined within a GitHub repository. It enables users to automate tasks such as building, testing, and deploying software directly within the GitHub platform, providing a powerful way to streamline development workflows.
+
+
+## Explanation of the diagram
+
+The diagram illustrates a workflow for building and pushing an image to the GitHub Container Registry. It outlines the steps involved in this process, including the creation of three specific tags: pull request number, pull request name, and commit SHA based on the commit made in a pull request. Once the image is in the GCR, a Kubernetes deployment can use the image
+
+## Diagram
+
+![Github image workflow](img/gh_docker_workflow.png)
```

Comment:
The information (emails + agency logo) needs to be at bottom of diagram.

Reply:
Yeah, I took the picture before applying the signature of the CFIA. If it's that important, I will need to remake the diagram from scratch...

Reply:
the source of the diagram should also be part of the commit to allow future edits

Reply:
In that case, I think we should'nt use excalidraw and use something else like Mermaid or D2

Reply:
Done. The signature is at the bottom of the diagram + I added a backup for the ansible and ghcr_docker_workflow diagram